server/bundler/loader.js

```diff
@@ -26,7 +26,12 @@ function getSectionsModule( sections ) {
 			} );
 		} );
 	} else {
-		dependencies = "var controller = require( 'controller' );\n";
+		dependencies = [
+			"var page = require( 'page' ),",
+			"\tcontroller = require( 'controller' ),",
+			"\preloadHub = require( 'sections-preload' ).hub;\n"
```

Comment:
To be honest, I'm not sure what it is for. However, whether code splitting is disabled/enabled, `'preloadHub.on( \'preload\', preload );',` is called and throws error because `preloadHub` was not found.

I can make two variants of [this](https://github.com/Automattic/wp-calypso/blob/master/server/bundler/loader.js#L33) so `preloadHub` won't be called.